worker/snapshot.go

```diff
@@ -242,6 +228,49 @@ func doStreamSnapshot(snap *pb.Snapshot, out pb.Worker_StreamSnapshotServer) err
 		return err
 	}
 
+	// Type and Schema keys always have a timestamp of 1. They all need to be sent
+	// with the snapshot. The stream framework does not send it when streaming at SinceTs > 1.
+	if stream.SinceTs > 0 {
+		buf := z.NewBuffer(10*MB, "DoStreamSnapshot")
+		defer buf.Release()
+		txn := pstore.NewTransactionAt(1, false)
+		defer txn.Discard()
+		copyKvsForPrefix := func(prefix []byte) error {
+			iopts := badger.DefaultIteratorOptions
+			iopts.Prefix = prefix
+			itr := txn.NewIterator(iopts)
+			defer itr.Close()
+
+			kv := bpb.KV{}
+			for itr.Rewind(); itr.Valid(); itr.Next() {
+				kv.Reset()
+				item := itr.Item()
```

Comment:
Since we finish using `item` before we call `itr.Next()`, it should be safe to use `item.Key()` and `item.Value()` (i.e. without copying).

Reply:
Thanks. Done.